config/routes.rb

```diff
@@ -9,6 +9,9 @@
     delete "/logout", to: "sessions#destroy"
     get 'password_resets/new'
     get 'password_resets/edit'
+    get "/register", to: "register#new"
```

Comment:
line 12 -> 14, you should define the clear router name for course: `resources :courses`